fsspec/implementations/dbfs.py

```diff
@@ -219,6 +226,60 @@ def _open(self, path, mode="rb", block_size="default", **kwargs):
         """
         return DatabricksFile(self, path, mode=mode, block_size=block_size, **kwargs)
 
+    def retry_request(func, retries=5):
+
+        RETRIABLE_EXCEPTIONS = (
+            requests.exceptions.ChunkedEncodingError,
+            requests.exceptions.ConnectionError,
+            requests.exceptions.ReadTimeout,
+            requests.exceptions.Timeout,
+            requests.exceptions.ProxyError,
+            requests.exceptions.SSLError,
+            requests.exceptions.ContentDecodingError,
```

Comment:
Do these capture things like socket timeout and DNS lookup timeout? Maybe not a great need to be too comprehensive at this point.

Would be requests.exceptions.RequestException (which includes all of the above) be too general?

Reply:
I think they only capture Http requests timeouts, all of the above (and more) inherit from requests.exceptions.RequestException, should we retry on all request exceptions (I just kept the ones from the code U share)? I found the issue only with the 429 error, it doesn't make sense to retry on all the request expectations, we should agree which ones to retry

Reply:
I'm not sure! It probably does *not* make sense to retry everything, some errors are permanent. I am happy to be conservative and add extra cases to the list later, if they become necessary.

Reply:
OK, lets keep the ones already there as they are the same included in the gcsfs implementation (without the aiohttp.client_exceptions.ClientError)